applications/serial_lte_modem/src/gnss/slm_at_gnss.c

```diff
@@ -652,6 +660,7 @@ static void fix_rep_wk(struct k_work *work)
 	}
 
 update_pgps:
+#if defined(CONFIG_SLM_PGPS)
```

Comment:
@junqingzou, question about the lines 642-644 "if (IS_ENABLED(CONFIG_SLM_LOG_LEVEL_DBG))....", Why the goto-jump is dependent on the log level? It is hard to understand.